docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
+NVIDIA GPU. This is achieved by configuring the underlying MicroK8s, that the
+DSS is relying for running the containerised workloads.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+* Have run the :ref:`tutorial` tutorial
+
+Install the NVIDIA Operator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To ensure the DSS can utilise the NVIDIA GPUs:
+
+1. The host will need to have the NVIDIA drivers installed
+2. MicroK8s will need to be setup to utilise those drivers
```

Comment:
```suggestion
2. MicroK8s must be set up to utilise NVIDIA drivers.
```
Adding links to these two points for further instructions on how to do theses things are definitely useful.

Reply:
Yes! I'll try to add a link to microK8s docs for this